phalcon/db/adapter/pdo.zep

```diff
@@ -85,9 +85,12 @@ abstract class Pdo extends Adapter
 	 */
 	public function connect(descriptor = null)
 	{
-		var username, password, dsnParts, dsnAttributes,
+		var eventsManager,
+			username, password, dsnParts, dsnAttributes,
 			persistent, options, key, value;
 
+		let eventsManager = <ManagerInterface> this->_eventsManager;
```

Comment:
I think that we should try to keep the current way of being able to use the normal EM or creating a new one.